cavalier_contours/src/macros.rs

```diff
@@ -9,8 +9,8 @@ macro_rules! assert_fuzzy_eq {
                 if !(left_val.fuzzy_eq(*right_val)) {
                     panic!(
                         r#"assertion failed: `left.fuzzy_eq(right)`
-  left: `{:?}`,
- right: `{:?}`"#,
+                          left: `{:?}`,
+                         right: `{:?}`"#,
```

Comment:
Don't think we want this indented as it's inside raw string literal so it will format weird (these lines indented significantly while others are not).